src/python/deepgnn/graph_engine/data/cora.py

```diff
@@ -102,7 +105,13 @@ def _load_raw_graph(
     parser.add_argument("--data_dir", default=f"{tempfile.gettempdir()}/cora", type=str)
     parser.add_argument("--train_node_ratio", default=1.0, type=float)
     parser.add_argument("--random_selection", action="store_true")
+    parser.add_argument("--num_partitions", default=1)
     args = parser.parse_args()
 
-    g = CoraFull(args.data_dir, args.train_node_ratio, args.random_selection)
-    print(f"graph data: {args.data_dir}")
+    g = CoraFull(
+        args.data_dir,
+        args.train_node_ratio,
+        args.random_selection,
+        num_partitions=args.num_partitions,
+    )
+    print(f"graph data: {args.data_dir} n_partitions {args.num_partitions}")
```

Comment:
nit: remove n_partitions output, we don't log train ratio or random selection neither.